src/ship_commands.cpp

```diff
@@ -8,14 +8,14 @@
 
 
 
-static inline float fmin( float a, float b )
-{
-    return (a < b) ? a : b;
-}
-static inline float fmax( float a, float b )
-{
-    return (a > b) ? a : b;
-}
+// static inline float fmin( float a, float b )
+// {
+//     return (a < b) ? a : b;
+// }
+// static inline float fmax( float a, float b )
```

Comment:
Similarly here. Either conditionally based on a configure check, or remove if widely supported already.

Some consideration must be given to windows, though. MSVC lacked fmin/fmax a while ago, I'm not sure whether it includes it now in its math templates.

Reply:
I would expect that these are widely supported by this point. I think I will go ahead and remove them.

Reply:
Removed.